5._Governance.md

```diff
@@ -14,18 +14,19 @@ The Project includes the following roles. Additional roles may be adopted and do
 
 **1.2. Contributors.** "Contributors" are those Participants that have (a) made Contributions to the Project subject to the Community Specification License; and/or (b) contributed code or documentation to Project assets other than specifications.
 
-**1.3. Maintainers.** "Maintainers" are responsible for organizing activities around developing, maintaining, and updating the specification(s) and other assets developed by the Project. Maintainers are also responsible for determining consensus and coordinating appeals. Examples of the responsibility of a Maintainer (directly or by delegation) include:
-
-* leading workstream meetings and tracking progress
-* setting the agenda, scheduling meetings and keeping minutes
-* actively engaging with the proposals process, both in GitHub issues and the slsa-proposals repo
-* triaging and prioritizing issues
-* contributing and reviewing pull requests
-* manage the day-to-day planning, operation, organization, deliverables and alignment with other workstreams
-* coordinating efforts with the Steering Committee, other workstreams and other external projects
-* ensuring that the contents of their workstream's materials accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines
-
-A Contributor may become a Maintainer with the Approval of the Steering Committee. A Maintainer may be removed with the Approval of the Steering Committee. A Steering Committee Member may not deliberate or vote on their own appointment or removal as a Maintainer.
+**1.3. Maintainers.** "Maintainers" are responsible for reviewing and merging all proposed changes, and they guide the overall technical direction of the Project within the guidelines established by the Steering Committee. The specification's Maintainers are listed in [MAINTAINERS.md](https://github.com/slsa-framework/slsa/blob/main/MAINTAINERS.md), while other workstreams may have their own list of Maintainers.
```

Comment:
> while other workstreams may have their own list of Maintainers.

Would these just be at the root level in a git repo for the workstream?

Reply:
Perhaps workstreams isn't the correct term here? It may be simpler/clearer to just state

> , while other repositories may have their own list of Maintainers.

Reply:
Would the assumption then be that https://github.com/slsa-framework/slsa/blob/main/MAINTAINERS.md are the default maintainers for the organization unless otherwise specified in the specific repository?

Reply:
Yes. I tried to clarify. Please take a look and let me know what you think.

Reply:
That works for me, thanks!